lib/Dialect/TritonGPU/Transforms/WSLowering.cpp

```diff
@@ -230,8 +194,8 @@ void lowerTokenOperations(Operation *parentOp, int numCTAs,
 
       Value barrierEmptyView = builder.create<ttg::MemDescSubviewOp>(
           loc, singleBarrierMemDescType, bufferEmptyArray, idx);
-      unsigned bufferEmptyCount = numCTAs;
-      builder.create<ttng::InitBarrierOp>(loc, barrierEmptyView, numCTAs);
+      builder.create<ttng::InitBarrierOp>(loc, barrierEmptyView,
+                                          THREADS_PER_TASK);
```

Comment:
Can't comment on unchanged code. So for this later section
  // Insert a cluster barrier before the kernel exits. Without this barrier,
  // mbarrier_remote_arrive will fail if the remote CTA already exits.
Is it still valid with this change?

Reply:
Yeah, it would fail I guess. I'm getting rid of remote CTA mode as I'm not sure how it would be used. Let me know if I'm missing anything.